<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomMakeEditVirtualForm_template</class>
 <widget class="QDialog" name="RoomMakeEditVirtualForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="The name of the rooms dialog">Make/edit virtual room</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="setsLabel">
         <property name="text">
          <string comment="Sets of real rooms">Sets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="setsListWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="addSetPushButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeSetPushButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="selectedLabel">
         <property name="text">
          <string comment="Selected real rooms in set">Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="selectedRealRoomsListWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="clearPushButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="allLabel">
         <property name="text">
          <string comment="All real rooms">All real rooms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="allRealRoomsListWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="allPushButton">
         <property name="text">
          <string comment="Add all real rooms">All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
